core/domain/question_services.py

```diff
@@ -33,7 +33,7 @@ def _create_new_question(committer_id, question, commit_message):
         question: Question. question domain object.
         commit_message: str. A description of changes made to the question.
     """
-    question.validate()
+    question.validate(strict=True)
```

Comment:
Just wondering, do you need strict vs non-strict if we never save drafts?

Reply:
Yeah, we are only doing strict validation now, will remove this.

Reply:
Done.